fares-data-build-tool/fdbt-site
# feat: allow users to define the size of a group

# Description

-   This PR allows users to define a total size of their group for a group ticket.

# Testing instructions

-   Pull down this branch and run the site locally. Navigate to the /groupSize page via the URL and check that it behaves as expected.

# Type of change


# Checklist:


<img width="1680" alt="Screenshot 2020-07-29 at 09 56 27" src="https://user-images.githubusercontent.com/55277550/88779630-e1f2e600-d181-11ea-8954-b673d6532d5d.png">
